runtime/mangata-kusama/src/xcm_config.rs

```diff
@@ -291,7 +193,7 @@ pub type XcmRouter = (
 
 impl pallet_xcm::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type SendXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
+	type SendXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, ()>;
```

Comment:
similar to statemine, i though it would be useful to also disable arbitrary xcm calls
even though we have them filtered in XcmExecuteFilter

same change for rococo